.eslintrc.js

```diff
@@ -44,17 +44,18 @@ module.exports = {
     "jsdoc/require-param-type": "warn",
     "jsdoc/require-returns-description": "warn",
     "jsdoc/require-returns-type": "warn",
+    "mozilla/balanced-listeners": "error",
+    "mozilla/no-aArgs": "warn",
+    "mozilla/use-chromeutils-import": "off", // TODO: "warn"?
```

Comment:
I think this was a rule getting pulled in via an upgrade to latest "eslint-plugin-mozilla", so I silenced it for now and added a TODO to reassess later.